src/main/java/dbpath/DBPathConnectionHandler.java

```diff
@@ -60,4 +61,16 @@ public void closeConnection() {
         }
         connection = null;
     }
+
+    public void deleteDatabaseFolder() {
+        SqlSandboxUtils.deleteDatabaseFolder(dbPath.getDatabaseFile());
+    }
+
+    public boolean createDatabaseFolder() {
+        return new File(dbPath.getDatabaseFile()).getParentFile().mkdirs();
+    }
+
+    public String getDatabaseFileForLoggingPurposes() {
```

Comment:
the reason this is so specific is that you don't think we'll ever need just the raw path / if we do we're breaking the abstraction?